templates/marriage.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * @var System $system
+ * @var User $player
+ * @var string $self_link
+ * @var false|array $proposal_user
+ * @var array $spouse
+ */
+?>
+
+<style>
+    label {
+        width: 9em;
+        display: inline-block;
+    }
+    input[type=submit] {
+        margin: 3px;
+    }
+
+    div#perk_details {
+        width: 70%;
+
+        margin-left: 9em;
+        padding: 5px;
+
+        border: 1px solid black;
+    }
+</style>
+
+<table class="table">
+    <?php if($player->spouse <= 0): ?>
+        <?php if($player->spouse < 0): ?>
+            <?php if($proposal_user == false): ?>
+                <tr><th>Cold Feet!</th></tr>
+                <tr>
+                    <td>
+                        Your prospective spouse got cold feet! You are free to look for love again.
+                    </td>
+                </tr>
+            <?php else: ?>
+                <tr><th><?=$proposal_user['user_name']?>'s Proposal</th></tr>
+                <tr>
+                    <td style="text-align: center;">
+                        <form style='display:inline;' action="<?=$self_link?>" method="post">
+                            <input type="submit" name="accept_proposal" value="Accept" />
+                        </form>
+                        <form style='display:inline;' action="<?=$self_link?>" method="post">
+                            <input type="submit" name="deny_proposal" value="Deny" />
+                        </form>
+                    </td>
+                </tr>
+            <?php endif ?>
+        <?php else: ?>
+            <tr><th>Send a Proposal</th></tr>
+            <tr>
+                <td style="text-align: center; padding: 5px;">
+                    <form action="<?=$self_link?>" method="post">
+                        <label>Username:</label><input type="text" name="user_name" /><br />
+                        <input type="submit" name="propose" value="Propose" />
+                    </form>
+                </td>
+            </tr>
+        <?php endif ?>
+    <?php else: ?>
+        <tr><th>Marriage Detail</th></tr>
+        <tr>
+            <td style="padding: 8px;">
+                <label>Spouse: </label><a href="<?=$system->links['members']?>&user=<?=$player->spouse_name?>"><?=$player->spouse_name?></a><br />
+                <label>Anniversary:</label><?=Date('F j, Y', $player->marriage_time)?><br />
+                <label>Location:</label><?=$spouse['location']?><br />
+                <label>Yen:</label><?=number_format($spouse['money'])?><br />
+                <label>Ancient Kunai:</label><?=number_format($spouse['premium_credits'])?><br />
```

Comment:
Hmm what's the reason for showing this?

Reply:
Just thought it could encourage more cooperative game play. Suppose it could be just as easy to remove it and allow this communication through chat/PM.